server/routes/taskList.js

```diff
@@ -35,14 +38,15 @@ module.exports = ({ prisonerService, licenceService, audit }) => router => {
       const licenceStatus = getLicenceStatus(licence)
       const allowedTransition = getAllowedTransition(licenceStatus, req.user.role)
       const { statusLabel } = getStatusLabel(licenceStatus, req.user.role)
+      let taskListModel
+      const errorCodes = await caService.getReasonForNotContinuing(bookingId, res.locals.token)
```

Comment:
this should really only be called if `licenceStatus.stage === 'ELIGIBILITY' && req.user.role === 'CA'`.
It's quite a heavy weight call

Reply:
ok